demopacket.go

```diff
@@ -12,7 +12,7 @@ import (
 
 // NetMessageCreator creates additional net-messages to be dispatched to net-message handlers.
 //
-// See also: ParserConfig.AdditionalNetMessageCreators & Parser.RegisterNetMessageHandler()
+// See also: ParserConfig.AdditionalNetMessageCreators & parser.RegisterNetMessageHandler()
```

Comment:
Should stay upper-case otherwise we reference the private type as only public types are in the godoc.

```suggestion
// See also: ParserConfig.AdditionalNetMessageCreators & Parser.RegisterNetMessageHandler()
```